shared/feature/feeding/build.gradle.kts

```diff
@@ -0,0 +1,28 @@
+plugins {
+    id("AnimealPlugin")
+    id("com.android.library")
+    id("kotlin-kapt")
+}
+
+android {
+    namespace = "com.epmedu.animeal.feeding"
+}
+
+dependencies {
+    implementation(projects.library.common)
+    implementation(projects.library.extensions)
+    implementation(projects.library.foundation)
+    implementation(projects.library.resources)
+    implementation(projects.library.geolocation)
+
+    implementation(libs.androidx.datastore)
```

Comment:
I think there is no need for datastore, material.dialog.core or kapt plugin.
Please delete them.